.github/workflows/python-api-flair-cd.yaml

```diff
@@ -0,0 +1,50 @@
+name: flair-docker-cd
+on:
+  push:
+    branches:
+      - main
+    paths:
+      - "docker_images/flair/**"
+jobs:
+  build:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Set up Python ${{ matrix.python-version }}
+        uses: actions/setup-python@v2
+        with:
+          python-version: "3.8"
+      - name: Checkout
+        uses: actions/checkout@v2
+      - name: Set up QEMU
+        uses: docker/setup-qemu-action@v1
+      - name: Set up Docker Buildx
+        uses: docker/setup-buildx-action@v1
+      - name: Install dependencies
```

Comment:
I don't think we need kubectl, helm kubernetes and uvicorn(?)

Reply:
Thanks for the catch !